src/psyclone/psyir/symbols/routinesymbol.py

```diff
@@ -61,12 +60,19 @@ def __init__(self, name, datatype=None, **kwargs):
         if datatype is None:
             datatype = NoType()
         super(RoutineSymbol, self).__init__(name, datatype)
```

Comment:
Since we are here, can you update this to super() to remove the pylint warning

Reply:
Done.